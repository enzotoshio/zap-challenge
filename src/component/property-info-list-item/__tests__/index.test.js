import React from 'react';
import renderer from 'react-test-renderer';

import PropertyInfoListItem from '../';

describe('PropertyInfoListItem', () => {
  it('increments the current page', () => {
    const item = {
      images: ['pumba'],
      pricingInfos: {
        businessType: 'sale',
        price: '10',
      },
      bathrooms: 10,
      bedrooms: 10,
      usableAreas: 10,
      parkingSpaces: 10,
    };
    const component = renderer.create(
      <PropertyInfoListItem {...item}>
        <span />
      </PropertyInfoListItem>
    );

    const dom = component.toJSON();

    expect(dom).toMatchSnapshot();
  });
});
