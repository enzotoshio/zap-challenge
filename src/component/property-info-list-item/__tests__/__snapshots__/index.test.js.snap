// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PropertyInfoListItem increments the current page 1`] = `
<div
  className="product-info-list-item"
>
  <div
    className="thumbnail"
  >
    <div
      className="thumbnail-arrow-container thumbnail-left-arrow-container "
      onClick={[Function]}
    >
      <div
        className="thumbnail-arrow thumbnail-left-arrow"
      />
    </div>
    <img
      alt="Fotos do imóvel"
      src="pumba"
    />
    <div
      className="thumbnail-arrow-container thumbnail-right-arrow-container "
      onClick={[Function]}
    >
      <div
        className="thumbnail-arrow thumbnail-right-arrow"
      />
    </div>
  </div>
  <div
    className="description"
  >
    <p>
      Apartamento para
       
      venda
    </p>
    <p>
      Banheiros: 
      10
    </p>
    <p>
      Quartos: 
      10
    </p>
    <p>
      Área: 
      10
      m
      <sup>
        2
      </sup>
    </p>
    <p>
      Vagas: 
      10
    </p>
    <p>
      R$
      10
    </p>
  </div>
</div>
`;
